packages/web/components/Technology/Details/Tabs/FAQ/index.js

```diff
@@ -1,69 +1,96 @@
-import React from 'react';
+import React, { useState } from 'react';
+import useSWR from 'swr';
 import { useForm } from 'react-hook-form';
 import { FiSend } from 'react-icons/fi';
+import { useTechnology, useAuth } from '../../../../../hooks';
+import Loading from '../../../../Loading';
 import { toast } from '../../../../Toast';
 import Section from '../../Section';
 import { InputField } from '../../../../Form';
 import Question from './Question';
+import {
+	getTechnologyQuestions,
+	createTechnologyQuestion,
+} from '../../../../../services/technology';
 import * as Layout from '../../../../Common/Layout';
 import * as S from './styles';
 
-const questionsMock = [
-	{
-		id: 1,
-		question: 'Boa noite, vocês auxiliam na instalação?',
-		answer: 'Boa noite. Sim, auxiliamos.',
-		created_at: '2020-11-09 12:53:24.000000',
-	},
-	{
-		id: 2,
-		question: 'Boa tarde, vocês auxiliam na instalação?',
-		answer: 'Boa tarde. Sim, auxiliamos.',
-		created_at: '2020-11-19 12:53:24.000000',
-	},
-	{
-		id: 3,
-		question: 'Bom dia, vocês auxiliam na instalação?',
-		answer: 'Bom dia. Sim, auxiliamos.',
-		created_at: '2020-03-14 12:53:24.000000',
-	},
-];
-
 const FAQ = () => {
 	const form = useForm();
+	const { technology } = useTechnology();
+	const { user } = useAuth();
+	const isLoggedIn = !!user?.email;
+	const [itemsCount, setItemsCount] = useState(5);
+
+	const {
+		data: { questions, totalItems },
+		isValidating,
+		mutate,
+	} = useSWR(
+		['get-technology-answered-questions', technology.id, itemsCount],
+		(_, id, perPage) =>
+			getTechnologyQuestions(id, {
+				perPage,
+				page: 1,
+			}),
+		{
+			initialData: [],
+			revalidateOnMount: true,
+		},
+	);
+
+	const onSubmit = async ({ question }) => {
+		const response = await createTechnologyQuestion({
+			technology: technology.id,
+			question,
+		});
 
-	const onSubmit = () => {
-		toast.success('Pergunta enviada com sucesso');
+		if (response) {
+			toast.success('Pergunta enviada com sucesso');
+		} else {
+			toast.error('Houve um erro ao enviar sua pergunta');
+		}
+
+		form.reset();
 	};
 
 	const loadMoreQuestions = () => {
-		toast.error('Carregar mais 10 perguntas e respostas');
+		setItemsCount(itemsCount + 5);
+		mutate('get-technology-answered-questions');
```

Comment:
You don't need to pass key here as this mutate is already bound to that query

See https://swr.vercel.app/docs/mutation#bound-mutate